### ISSUE TITLE
Feature request to add refresh/update link

### ISSUE THREAD

**Issue by nayzawoo** [agalwood/Motrix#286]
I want to request to add refresh/update link feature for expire download link like XDM.
And please add title on action buttons.
I really love this app! ❤️

**Comment by agalwood** [comment]
There is already a related issue. It is Chinese that you may not understand haha.
https://github.com/agalwood/Motrix/issues/266

To be developed.

**Comment by theweavrs** [comment]
@agalwood how can we go about doing this? Can the [`changeUri`](https://aria2.github.io/manual/en/html/aria2c.html#aria2.changeUri) method work for this?

**Comment by agalwood** [comment]
@theweavrs Yes, the task details have a lot of features to develop. In addition to [changeUri](https://aria2.github.io/manual/en/html/aria2c.html#aria2.changeUri) and [changeOption]().

But I think the beautiful typography of the UI is a problem, we need to redesign the dialog of the task detail.

**Comment by theweavrs** [comment]
@agalwood the task detail dialog should have the url of the task and after it an edit icon. Clicking the edit icon edit the url right there. Using this approach we wouldn't have to add any additional dialogs etc.

The details also should have the Path of the downloading file with the ability to change it (if possible).

**Comment by yuvrajraghuvanshis** [comment]
Any update on this?

**Comment by bharath619** [comment]
any update?

**Comment by abrattic** [comment]
This feature would be excellent to have.

**Comment by ameyjoshi3068** [comment]
Any update?

**Comment by agalwood** [comment]
Consolidating the expired-link refresh/update request into the current v2 discussion in #1788.
